src/main/java/com/ufcg/es5/BackendComplexoEsportivoUFCG/application/reservation/service/ReservationServiceImpl.java

```diff
@@ -132,4 +129,21 @@ private Reservation makeReservation(
         );
     }
 
+    private void checkReservationTimeValidity(Reservation reservation) {
+        LocalDateTime now = LocalDateTime.now();
+
+        if (reservation.getStartDateTime().isBefore(now.plusHours(24))) {
+            throw new SaceForbiddenException(
+                    ReservationExeceptionMessages.RESERVATION_PERMISSION_DENIED
```

Comment:
criar uma mensagem RESERVATION_CANCELLATION_TIME_EXPIRED = "Cancellation time for reservation has expired."